### PR TITLE
Fix documentation reader

### PR DESCRIPTION
**Pull request by michael**

### CODE REVIEW

**Comment by michael** [comment]
Could need a bit of more love in terms of providing all config via configurator pattern, but will do for a beta 4 release I think. Please review and merge.

File: packages/prose-editor/ProseEditor.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ ProseEditor.Prototype = function() {
   };
 
   this._initialize = function(props) {
-    var configurator = this.props.configurator;
```
Comment (obuchtala) [inline packages/prose-editor/ProseEditor.js:57]: whoa.. des hat ned kracht die ganze zeit?

Comment (obuchtala) [inline packages/prose-editor/ProseEditor.js:57]: na... what is the difference to this.props?

**Comment by obuchtala** [comment]
Fine with me to merge.. Though I probably need more exposure to grasp everything.
I.e. regarding generlizing TOCProviders.

File: packages/prose-editor/ProseEditor.js (hunk 1 above)
Comment (michael) [inline packages/prose-editor/ProseEditor.js:57]: Ja es kracht bei willReceiveProps weil da nextProps reinkommen. Drum is ned aufgfallen.

**Comment by michael** [comment]
Pushing the button...

**Pull request merged**